open_event/api/formats.py

```diff
@@ -0,0 +1,37 @@
+from flask.ext.restplus import Resource, Namespace, fields
+
+from open_event.models.session import Format as FormatModel
+from open_event.models.event import Event as EventModel
+from .helpers import get_object_list, get_object_or_404, get_object_in_event
+
+api = Namespace('formats', description='formats', path='/')
+
+format_ = api.model('Format', {
+    'id': fields.Integer(required=True),
+    'name': fields.String,
+    'label_en': fields.String,
+})
+
+
+@api.route('/events/<int:event_id>/formats/<int:id>')
+@api.response(404, 'Format not found')
+@api.response(400, 'Object does not belong to event')
+class Format(Resource):
+    @api.doc('get_format')
+    @api.marshal_with(format_)
+    def get(self, event_id, id):
+        """Fetch a format given its id"""
+        return get_object_in_event(FormatModel, id, event_id)
+
+
+@api.route('/events/<int:event_id>/formats')
+@api.param('event_id')
+class FormatList(Resource):
+    @api.doc('list_formats')
+    @api.marshal_list_with(format_)
+    def get(self, event_id):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2620934160/issues/source?bid=3233864&fileBranchId=3314204#l32)